docs/source/fetching-queries.md

```diff
@@ -118,4 +118,24 @@ By default, Apollo constructs queries and sends them to your graphql endpoint us
 
 If you want Apollo to use `GET` instead, pass `true` to the optional `useGETForQueries` parameter when setting up your `HTTPNetworkTransport`. This will set up all queries conforming to `GraphQLQuery` sent through the HTTP transport to use `GET`. 
 
->Please note that this is a toggle which affects all queries sent through that client, so if you need to have certain queries go as `POST` and certain ones go as `GET`, you will likely have to swap out the `HTTPNetworkTransport`. 
\ No newline at end of file
+>Please note that this is a toggle which affects all queries sent through that client, so if you need to have certain queries go as `POST` and certain ones go as `GET`, you will likely have to swap out the `HTTPNetworkTransport`.
+
+## JSON serialization
+
+The classes generated by Apollo iOS can be converted to JSON using their `jsonObject` property. This may be useful for conveniently serializing GraphQL instances for storage in a database, or a file.
+
+For example:
+```swift
+apollo.fetch(query: HeroAndFriendsNamesQuery(episode: .empire)) { (result, error) in
+  guard let data = result?.data else { return }
+
+  // Serialize the response as JSON
+  let json = data.jsonObject
+  let serialized = try! JSONSerialization.data(withJSONObject: json, options: [])
+  
+  // Deserialize the response
+  let deserialized = try! JSONSerialization.jsonObject(with: serialized, options: []) as! JSONObject
+  let heroAndFriendsNames = try! HeroAndFriendsNamesQuery.Data(jsonObject: deserialized)
+}
+```
+```
```

Comment:
looks like an extra set of backticks snuck in here

Reply:
Thanks for catching that! It's fixed now